prow/scripts/cluster-integration/kyma-gke-upgrade.sh

```diff
@@ -344,40 +346,59 @@ function checkTestPodTerminated() {
     return 1
 }
 
-createTestResources() {
-    shout "Create e2e upgrade test resources"
-    date
+function installTestChartOrFail() {
+  local path=$1
+  local name=$2
+  local namespace=$3
 
-    injectTestingAddons
+  shout "Create ${name} resources"
+  date
 
-    if [  -f "$(helm home)/ca.pem" ]; then
-        local HELM_ARGS="--tls"
-    fi
+  local HELM_ARGS
+  if [[ -f "$(helm home)/ca.pem" ]]; then
+      HELM_ARGS="--tls"
```

Comment:
Is migration to Helm3 a really simple task that can be done here? Just a thought only. I won't block the PR.

Reply:
As discussed offline, we will keep it like that for now.

Reply:
Not applicable until we release 1.15